modules/ExternalPlayers/resources/mw.EmbedPlayerYouTube.js

```diff
@@ -410,6 +410,12 @@
 				callback();
 				if( mw.getConfig('autoPlay') || _this.isPlaylist){
 					if (mw.isMobileDevice()){
+						if (mw.isIphone()){
+							$(".largePlayBtn").hide();
```

Comment:
I think we need a better way referring components instead of direct jquery CSS queries.